lib/Styled/Icon.tsx

```diff
@@ -165,10 +162,10 @@ export const StyledIcon = styled(Icon)<IStyledIconProps>`
   ${props =>
     props.displayInline &&
     `
-  display: inline; 
+  display: inline;
   vertical-align: middle;
   `}
-  
+
```

Comment:
Should we add a `display: block` option, it was a default option when `displayInline` wasn't specified in props. 

```css
  ${props =>
    !props.displayInline &&
    `
  display: block
  `}
```

Reply:
Along with above, when vertically centering elements it might be good to put following to outer element (in this case button).
```css
display: flex;
align-items: center;
```
With that, we should get pixel-perfect center without having to think if icon is `block` or `inline` (currently it is slight moved down)

Reply:
Done!